### ISSUE TITLE
[PROJECT] Move embedded learning-area examples into content

### ISSUE THREAD

**Issue by bsmth** [mdn/mdn#605]
As part of an initiative to reduce repo maintenance & code duplication, we'll be moving external examples into content. Some automation work to speed this up is done on https://github.com/mdn/content/compare/main...bsmth:content:move-learning-area-examples, but to avoid one large PR which is not detail-focused, let's take our time to make sure the content looks and works great in smaller pieces.

__How to help (en-US `mdn/content`):__

* Go to the [tracking sheet](https://docs.google.com/spreadsheets/d/1-djSUIsEfIVlPuv9IgXfjtQqq9ZSNAm4nu09LE-lapE/edit?gid=0#gid=0)
* Pick between 5 and 10 files and mark yourself as working on some in the sheet
* Grab changes from https://github.com/bsmth/content/tree/move-learning-area-examples

Apply changes to your fork like this:

```bash
# Add the bsmth/content fork as a remote
git remote add bsmth https://github.com/bsmth/content.git
git fetch bsmth

# Start from your fork's main branch
git checkout main && git fetch && git pull
# Create a branch "my-examples-work" or whatever you want to call it from main
git branch my-examples-work && git checkout my-examples-work

# Check out specific files from my fork
git checkout bsmth/move-learning-area-examples -- path/to/example-file path/to/example-file2
```

At this point, you should have changes ready to be tidied up, and you can open a PR when ready.

__Checklist:__

* The feature works as expected, compared to the original
* The example is as short as possible with no unused styles or HTML
* Images use the https://github.com/mdn/shared-assets repo (add missing images to shared assets if you need)
* The surrounding prose makes sense given the changes
* Your PR links to this issue
* __Don't__ copy any 'playable editor' code (`playable.js` & `styles.css`) or assets

If you're unsure, you can ask, or highlight things as unsure in the PR.

__How to help (Translated content):__

When a PR is merged, you can help by porting the changes from mdn/content@main to translated-content. This will be really important to make sure translated versions don't get left behind with this project.

__Why:__

There are a few reasons why we are moving the examples:

* Help guarantee code quality of examples in content
* Consistency in code editors / code pens in docs pages
* More examples work with the playground out-of-the box
* Minimize code duplication so there are fewer pull requests and reviews needed in the org
* Fewer repositories to maintain

__Tracking sheet:__ https://docs.google.com/spreadsheets/d/1-djSUIsEfIVlPuv9IgXfjtQqq9ZSNAm4nu09LE-lapE/edit?gid=1560347173#gid=1560347173

__Relates to:__

- [x] https://github.com/mdn/mdn/issues/597

**Comment by bsmth** [comment]
> [@bsmth](https://github.com/bsmth) these are interesting. To be honest, I'd not really thought about the remaining usage of `{{EmbedGHLiveSample()}}`. I'd be happy to work on replacing these with regular `{{EmbedLiveSample()}}` instances.

I guess that's 9 pages under `/en-us/learn_web_development/` to have a look at. For instance, here, maybe it's okay: https://developer.mozilla.org/en-US/docs/Learn_web_development/Core/Structuring_content/Test_your_skills/HTML_text_basics#task_1 (probably better than a screenshot of HTML)

**Comment by chrisdavidmills** [comment]
> > [@bsmth](https://github.com/bsmth) these are interesting. To be honest, I'd not really thought about the remaining usage of `{{EmbedGHLiveSample()}}`. I'd be happy to work on replacing these with regular `{{EmbedLiveSample()}}` instances.
> 
> I guess that's 9 pages under `/en-us/learn_web_development/` to have a look at. For instance, here, maybe it's okay: https://developer.mozilla.org/en-US/docs/Learn_web_development/Core/Structuring_content/Test_your_skills/HTML_text_basics#task_1 (probably better than a screenshot of HTML)

Nah, I've just replaced them with regular live samples: https://github.com/mdn/content/pull/40888.

This was really easy, as all the code required is already on the page. Took me about 15 minutes.

**Comment by bsmth** [comment]
Thanks, Chris! It's looking pretty good now, as of today, I only see the following left:

```bash
grep -roi "EmbedGHLiveSample.*learning-area" ./files
./files/en-us/web/css/_colon_valid/index.md:EmbedGHLiveSample("learning-area
./files/en-us/web/html/reference/elements/input/checkbox/index.md:EmbedGHLiveSample("learning-area
./files/en-us/learn_web_development/core/structuring_content/test_your_skills/advanced_html_text/index.md:EmbedGHLiveSample("learning-area
```
